geeksurvey/templates/footer.html

```diff
@@ -4,15 +4,8 @@
 -->
 <!-- Footer-->
 <footer class="mt-auto py-3 bg-dark text-center">
-  <p>
-    <a class="m-0 text-white-50 text-decoration-none" href="{% url 'working' %}"> Help </a>
-  </p>
-  <p>
-    <a class="m-0 text-white-50 text-decoration-none" href="https://ceias.nau.edu/capstone/projects/CS/2022/SuperGeeks/"> About </a>
-  </p>
-  <p>
-    <a class="m-0 text-white-50 text-decoration-none" href="https://github.com/NAU-SuperGeeks/geeksurvey"> Source Code </a>
-  </p>
-  <p class="m-0 " style="color:#666;"> Copyright &copy; GeekSurvey 2022 </p>
+    <p class="m-0"> <a class="text-white-50 text-decoration-none" href="{% url 'working' %}"> Help </a></p>
```

Comment:
:+1: didnt realize m-0 on these p tags was keeping the padding low. Looks better.

Reply:
Yup! It's a Bootstrap margin class :).

https://getbootstrap.com/docs/4.0/utilities/spacing/